src/page.es6

```diff
@@ -127,22 +126,24 @@
         return;
       }
     }
-  };
+  }
 
   // Internal: Compares the elements of two Arrays to check if they
   // are logically the same.
   //
   // Returns true or false.
-  Page.prototype._arrayMatches = function(one, two) {
+  _arrayMatches(one, two) {
     if (one.length != two.length) return false;
 
-    for (var i = 0; i < one.length; ++i) {
+    for (let i = 0; i < one.length; ++i) {
       if (one[i] !== two[i]) return false;
     }
 
     return true;
-  };
+  }
+}
 
-  root.page = new Page();
-  root.Page = Page;
-})(this);
+export default Page;
+if(window) {
```

Comment:
It does by defining the global `Page` object, but the `page` instance (a syntax sugar that we have here) needs to be handled manually.